Solution1/BusinessLayer/DTOs/ForecastWeatherDTO.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLayer.DTOs
+{
+    public class ForecastWeatherDTO
+    {
+        public List<WeatherForDate> WeatherValuesForPeriod { get; set; }
```

Comment:
use one naming either Weather either Weather Values, either for date, either for period